openfisca_web_api_preview/scripts/serve.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+
+import os
+import sys
+import argparse
+from gunicorn.app.base import BaseApplication
+from gunicorn.six import iteritems
+
+from openfisca_core.scripts import add_tax_benefit_system_arguments, detect_country_package
+from ..app import create_app
+
+
+DEFAULT_PORT = '5000'
+BASE_CONF = {
+    'bind': '%s:%s' % ('127.0.0.1', DEFAULT_PORT),
```

Comment:
Any reason not to use the more usual `'127.0.0.1:{}'.format(DEFAULT_PORT)` ?

Reply:
A bad taste for novelty? 😅 
Replaced.